cumulusci/cli/cci.py

```diff
@@ -65,13 +65,16 @@ def timestamp_file():
         yield f
 
 
-def get_latest_version():
+def get_latest_final_version():
```

Comment:
is "final" a PyPI term? nothing about this is...final...per se...

Reply:
It's from PEP 440: https://www.python.org/dev/peps/pep-0440/#final-releases